pkgs/top-level/all-packages.nix

```diff
@@ -19601,7 +19601,8 @@ in
 
     rtl8821cu = callPackage ../os-specific/linux/rtl8821cu { };
 
-    rtlwifi_new = callPackage ../os-specific/linux/rtlwifi_new { };
+    rtw88 = if lib.versionAtLeast kernel.version "4.14" then callPackage ../os-specific/linux/rtw88 { } else null;
```

Comment:
Changing a package to null is not allowed. Please set disabled, broken or similar.

Reply:
Done!